homeassistant/components/knx/climate.py

```diff
@@ -322,6 +354,43 @@ async def async_set_preset_mode(self, preset_mode: str) -> None:
             )
             self.async_write_ha_state()
 
+    @property
+    def fan_mode(self) -> str:
+        """Return the fan setting."""
+
+        fan_speed = self._device.current_fan_speed
+
+        if fan_speed is None:
+            return FAN_OFF
+
+        if fan_speed == 0:
+            return FAN_OFF
+
+        if self._attr_fan_modes is None:
+            return FAN_OFF
```

Comment:
```suggestion
        if not fan_speed or self._attr_fan_modes is None:
            return FAN_OFF
```
Those can be combined to a single `if`.
Do we need explicit check for `self._attr_fan_modes`? I think if it was `None`, `self._device.current_fan_speed` would also be `None` so this check would never be reached.

Reply:
yes, but mypy doesn't know it and complains about it. Is it fine to leave mypy warnings?

Reply:
Then just leave it in for type safety, that's fine. 
We can't ignore mypy warnings as it would not pass CI, but we can either add a check or add `# type: ignore[...]`